models/comments.models.js

```diff
@@ -31,3 +31,11 @@ exports.createComment = (body, username, article_id) => {
       return rows[0];
     });
 };
+
+exports.removeComment = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id])
+    .then(() => {
+      return Promise.resolve({ status: 204 });
```

Comment:
The the object you have passed in - {status: 204} - doesn't need to be here. All the Promise.resolve( ) will do is allow us when invoking the model to access the next then block where you are explicitly sending back a 204 status code anyway.